### DISCUSSION TITLE
前端状态机系列：SCXML与XState对应关系（Front-end state machine series: SCXML and XState correspondence）

### DISCUSSION THREAD

**Discussion by lecepin** [statelyai/xstate#2940] [Show and tell]
[前端状态机系列：SCXML与XState对应关系](https://github.com/lecepin/blog/blob/main/%E5%89%8D%E7%AB%AF%E7%8A%B6%E6%80%81%E6%9C%BA%E7%B3%BB%E5%88%97%EF%BC%9ASCXML%E4%B8%8EXState%E5%AF%B9%E5%BA%94%E5%85%B3%E7%B3%BB.md)

<img src="https://user-images.githubusercontent.com/11046969/149446534-3b7624d5-cd3c-4c0f-ac43-8621a18c45e2.png" width="400" />

**Comment by ChrisShank** [comment]
This is awesome! 🙌🏼

**Comment by nathanbowang** [comment]
Thanks a lot. This is the best tutorial I've seen.
